_layouts/project.html

```diff
@@ -4,10 +4,182 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>Projects{% if page.title %} | {{ page.title }}{% endif %}</title>
     {% include  assets.html %}
+    <link rel="stylesheet" href={{ "/css/project.css" | relative_url }} />
   </head>
   <body>
-    {% include header.html %}
-    <div class="page-content">{{ content }}</div>
-    <script type="text/javascript" src={{ "/js/project_processor.js" | relative_url }}></script>
+    <nav>
+      <div class="nav-wrapper blue">
+        <a href="/" class="brand-logo">NITDgpOS</a>
+        <a href="#" data-activates="mobile-nav" class="button-collapse">
+          <i class="fa fa-bars"></i>
+        </a>
+        <ul class="right hide-on-med-and-down">
+          <li><a href="https://github.com/nit-dgp/{{page.title}}">Get Involved!</a></li>
+        </ul>
+        <ul class="side-nav" id="mobile-nav">
+          <li><a href="https://github.com/nit-dgp/{{page.title}}">Get Involved!</a></li>
+        </ul>
+      </div>
+    </nav>
+    <div class="center main-content">
+      {% if page.logo %}{% endif %}
```

Comment:
Just a opening and closing `if`? Do you intend to do something with this page variable?